src/reducer/authReducer.js

```diff
@@ -0,0 +1,46 @@
+const authReducer = (state, dispatch) => {
+    console.log(state, dispatch);
```

Comment:
You can remove redundant logs from the final code.

Reply:
sure